[BUG] sdist is missing `mypy.ini`

### setuptools version

69.2.0

### Python version

3.9.18

### OS

OpenIndiana

### Additional environment information

_No response_

### Description

The sdist package at PyPI is missing the `mypy.ini` file and so the testing fails.

### Expected behavior

Tests pass.

### How to Reproduce

Download sdist for setuptools 69.2.0 and run tests.

### Output

```console
=================================== FAILURES ===================================
41: error: Module "setuptools.extern" has no attri
mypy exited with status 1.
68: error: Library stubs not installed for "pkg_re
1: error: Skipping analyzing "setuptools.extern.mo
8: error: Module "setuptools.extern" has no attrib
4: error: Skipping analyzing "setuptools.extern.ja
16: error: Skipping analyzing "setuptools.extern.j
6: error: Cannot find implementation or library st
19: error: Skipping analyzing "Cython.Distutils.bu
14: error: Skipping analyzing "setuptools.extern.m
79: error: Skipping analyzing "setuptools.extern.j
523: error: "Distribution" has no attribute "names
77: error: Cannot find implementation or library s
30: error: Module "setuptools.extern" has no attri
22: error: Skipping analyzing "setuptools.extern.m
10: error: Module "setuptools.extern" has no attri
210: error: Skipping analyzing "setuptools.extern.
129: error: Skipping analyzing "setuptools.extern.
119: error: "Distribution" has no attribute "inclu
35: error: Skipping analyzing "setuptools.extern.p
9: error: Skipping analyzing "setuptools.extern.pa
25: error: Skipping analyzing "setuptools.extern.m
24: error: Skipping analyzing "setuptools.extern.m
41: error: Skipping analyzing "setuptools.extern.m
8: error: Module "pkg_resources.extern" has no att
18: error: Skipping analyzing "setuptools.extern.p
6: error: Skipping analyzing "setuptools.extern.ja
4: error: Skipping analyzing "jaraco": module is i
12: error: Skipping analyzing "setuptools.extern.p
5: error: Skipping analyzing "jaraco.path": module
```

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -13,6 +13,7 @@ include MANIFEST.in
 include LICENSE
 include launcher.c
 include msvc-build-launcher.cmd
+include mypy.ini
 include pytest.ini
 include tox.ini
 include setuptools/tests/config/setupcfg_examples.txt

